nifi-docs/src/main/asciidoc/walkthroughs.adoc

```diff
@@ -0,0 +1,1099 @@
+* `$EDITOR conf/nifi.properties` -- Opens the `nifi.properties` file in a text editor
+* Update the following lines:
+** `nifi.remote.input.host=` -> `nifi.remote.input.host=secure.nifi`
+** `nifi.remote.input.secure=false` -> `nifi.remote.input.secure=true`
+** `nifi.remote.input.socket.port=` -> `nifi.remote.input.socket.port=10443`
+** `nifi.web.http.port=8080` -> `nifi.web.http.port=`
+** `nifi.web.https.host=` -> `nifi.web.https.host=secure.nifi`
+** `nifi.web.https.port=` -> `nifi.web.https.port=9443`
+** `nifi.security.keystore=` -> `nifi.security.keystore=./conf/keystore.jks`
+** `nifi.security.keystoreType=` -> `nifi.security.keystoreType=jks`
+** `nifi.security.keystorePasswd=` -> `nifi.security.keystorePasswd=passwordpassword`
+** `nifi.security.truststore=` -> `nifi.security.truststore=./conf/truststore.jks`
+** `nifi.security.truststoreType=` -> `nifi.security.truststoreType=jks`
+** `nifi.security.truststorePasswd=` -> `nifi.security.truststorePasswd=passwordpassword`
+. Generate the client certificate keystore from the client certificate and key.
+`openssl pkcs12 -export -out CN=my_username.p12 -inkey client.key -in client.pem` -- Creates the PKCS12 keystore containing the client certificate and private key. This command prompts for an *export password* twice
+
+From this point, the instructions are identical to those using the TLS Toolkit, starting at <<load-client-certificate-to-browser,Loading the client certificate in the browser>>.
+
+==== Securing NiFi with TLS Toolkit and Provided Intermediate CA
+
+Occasionally, an IT department will provide an intermediate CA certificate which is signed by a trusted certificate and can be used to generate server and client certificates on-demand by the NiFi admin without further intervention by the IT department. The TLS toolkit provides a mechanism to use the intermediate certificate to sign generated certificates.
+
+The end result will consist of a self-signed external CA (the root), an intermediate CA used to sign NiFi certificates, a keystore and truststore containing the necessary certificates for the NiFi instance to operate, and a client certificate for a human user to access NiFi.
+
+// TODO: Write
+CAUTION: Develop section
+
+== Deploying a NiFi Cluster
+
+Apache NiFi can run in either _standalone_ or _clustered_ mode. A standalone node is often sufficient for dataflow operations, but in a production or high-volume environment, a cluster is more performant and resilient. For more information, see link:administration-guide.html#clustering[NiFi Administrator's Guide: Clustering^].
+
+=== Creating a NiFi Cluster
+
+// TODO: Write
+CAUTION: Needs a diagram
+
+CAUTION: Develop section
+
+. Copy the NiFi run directory 3x
+. Configure the settings
+. Start each node
+
+// TODO: Write
+=== Creating and Securing a NiFi Cluster with the TLS Toolkit
+
```

Comment:
I realize this section is still under development, but perhaps add a similar table to what exists for the secured cluster version. It will give some context even to the brief 3 steps currently provided.